modules/util/util.go

```diff
@@ -124,3 +125,10 @@ func MergeInto(dict map[string]interface{}, values ...interface{}) (map[string]i
 
 	return dict, nil
 }
+
+// JSONEscape esapes a string for JSON output, replacing " and \
+func JSONEscape(input string) string {
+	bytes, _ := json.Marshal(input)
```

Comment:
Dropping the error may be fine for this particular input, but for an exported global util function not so much?

Reply:
My understanding is that `json.Marshal` can not error when passed a string so I think it's fine to ignore.

Reply:
Yes for a string deep diving this appears to be true.

My suspicion is that you should just put a comment above this to state it.

```suggestion
	// json.Marshal will never return an error when marshalling a string input.
	bytes, _ := json.Marshal(input)
```